support/util/performanceprofilestatus/performanceprofilestatus.go

```diff
@@ -0,0 +1,160 @@
+package performanceprofilestatus
+
+import (
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	performanceprofilev2 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/performanceprofile/v2"
+	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
+	"gopkg.in/yaml.v2"
+)
+
+func Available() *performanceprofilev2.PerformanceProfileStatus {
+	lastHeartbeatTime := "2024-04-18T06:55:45Z"
```

Comment:
Same as @enxebre mentioned before, for all regarding testing stub

Reply:
@jparrill All these testing stubs are needed to create a mocked config map that will hold the performance profile status. These stubs used to create the following [3 config maps](https://github.com/rbaturov/hypershift/blob/add-pp-status-cm/hypershift-operator/controllers/nodepool/nodepool_controller_test.go#L2386)  in the `nodepool_controller_test.go`.